styles.css

```diff
@@ -1,179 +1,293 @@
 :root {
-  --primary: hsl(226, 68%, 50%);
-  --secondary: hsl(225, 25%, 67%);
-  --success: hsl(123, 68%, 44%);
-  --failure: hsl(2, 88%, 44%);
-  --spacing-xs: 0.5rem;
-  --spacing-small: 0.75rem;
-  --spacing-md: 1rem;
-  --spacing-large: 1.5rem;
-  --spacing-xl: 2rem;
+	--primary: hsl(226, 68%, 50%);
+	--secondary: hsl(226, 58%, 67%);
+	--success: hsl(123, 68%, 44%);
+	--failure: hsl(2, 88%, 44%);
+	--white: hsl(0, 0%, 100%);
+	--activeBtn: hsl(271, 66%, 71%);
+	--spacing-xs: 0.5rem;
+	--spacing-small: 0.75rem;
+	--spacing-md: 1rem;
+	--spacing-large: 1.5rem;
+	--spacing-xl: 2rem;
 }
 
 body {
-  font-size: 16px;
-  font-family: "Roboto", sans-serif;
-  margin: 0;
+	font-size: 16px;
+	font-family: 'Roboto', sans-serif;
+	margin: 0;
 }
 
 ul,
 li {
-  padding: 0;
-  margin: 0;
-  list-style: none;
-  text-transform: uppercase;
+	padding: 0;
+	margin: 0;
+	list-style: none;
+	text-transform: uppercase;
 }
 
 a {
-  text-decoration: none;
+	text-decoration: none;
+}
+
+a:active,
+a:hover {
+	color: var(--activeBtn);
+}
+
+.sticky {
+	position: fixed;
```

Comment:
Do note that there is also newer, albeit less widely supported option of `sticky` 😉